### PR TITLE
[Draft] Sticky notes

### PR DESCRIPTION
**Pull request by philemone**
## Describe your changes

This is just a draft with stickyNotes functionality added 

## Checklist before merge
- [ ] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

## Release Notes

- **New Features**
  - Introduced sticky notes functionality, including adding, updating, and deleting sticky notes within scenarios.
  - Added a new `StickyNotesPanel` to the toolbar for easy access to sticky note features.
  - Implemented API endpoints for managing sticky notes, allowing CRUD operations through the UI.

- **Enhancements**
  - Improved UI rendering for sticky notes with dynamic styling and interaction capabilities.
  - Enhanced error handling and notifications for sticky note operations.

- **Documentation**
  - Updated API documentation to include new sticky notes endpoints and usage examples.

### CODE REVIEW

**Comment by bartektartanus** [comment]
@coderabbitai review

File: designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/stickynotes/StickyNoteEvent.scala [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package pl.touk.nussknacker.ui.api.description.stickynotes
+
+import io.circe.{Decoder, Encoder}
+
+object StickyNoteEvent extends Enumeration {
+  implicit val typeEncoder: Encoder[StickyNoteEvent.Value] = Encoder.encodeEnumeration(StickyNoteEvent)
+  implicit val typeDecoder: Decoder[StickyNoteEvent.Value] = Decoder.decodeEnumeration(StickyNoteEvent)
```
[the remark this answers is not in the document]
Comment (philemone) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/stickynotes/StickyNoteEvent.scala:7]: Thanks, I will stay with enumeration - it is fine in this use-case

**Comment by philemone** [comment]
@coderabbitai review newest change

**Comment by philemone** [comment]
@coderabbitai can you close comments which you think have been resolved by my recent changes and do a review one more time?

**Pull request closed without merging**